dotcom-rendering/src/web/components/Callout/Deadline.tsx

```diff
@@ -0,0 +1,50 @@
+import { css } from '@emotion/react';
```

Comment:
This file was added in a previous PR (https://github.com/guardian/dotcom-rendering/pull/6819) - as soon as it is merged into main this shouldn't show as a changed file.